### ISSUE TITLE
[indent] Requires not indenting long lists of generics

### ISSUE THREAD

**Issue by benwiles1** [typescript-eslint/typescript-eslint#1777]
**Repro**

```TS
export const onObj = aFunc<
    ATypeWithAReallyLongName,
    AnnotherTypeWithAReallyLongName,
    {
        anObjectThatIWasTooLazyToMakeAnAliasFor: number
    }
>({
    opt1: new ATypeWithAReallyLongName(),
    opt2: new AnnotherTypeWithAReallyLongName(),
})

```JSON
{
  "rules": {
    "@typescript-eslint/import": "error"
  }
}
```

**Expected Result**
That the generics are indented

**Actual Result**
The generics were forced to 0 indent

**Additional Info**
Perhaps I am being too picky, but I dont like that prettier prefers dense formatting, so I have been trying to use eslint for formatting. 

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `2.21.0` |
| `@typescript-eslint/parser`        | `2.21.0` |
| `TypeScript`                       | `3.7.5` |
| `ESLint`                           | `6.8.0` |
| `node`                             | `12.16.1` |
| `npm`                              | `6.13.4` |

**Comment by bradzacher** [comment]
duplicate of #455
